packages/manager/apps/telecom/src/app/telecom/telephony/line/softphone/addDevice/add-device.html

```diff
@@ -0,0 +1,111 @@
+<oui-modal
+    data-on-dismiss="$ctrl.goBack()"
+    data-heading="{{ ::'telephony_line_softphone_add_my_device' | translate }}"
+>
+    <p>
+        <b
+            data-translate="telephony_line_softphone_add_device_description_bold"
+        ></b>
+        <span
+            data-translate="telephony_line_softphone_add_device_description"
+        ></span>
+    </p>
+    <div
+        class="d-flex flex-wrap w-100 minw-0 align-items-center justify-content-between"
+    >
+        <form class="form-inline" name="$ctrl.softphoneAddDeviceForm">
+            <div class="row">
+                <oui-field
+                    class="col-md-12"
+                    data-help-text="{{:: 'telephony_line_softphone_generate_warning' | translate }}"
+                    label="{{ ::'telephony_line_softphone_add_device_label' | translate }}"
+                >
+                    <div class="input-group">
+                        <input
+                            type="text"
+                            id="input_name"
+                            name="input_name"
+                            class="oui-input"
+                            data-ng-model="$ctrl.model.name"
+                            data-ng-pattern="$ctrl.FIELD_NAME_REGEX"
+                            data-ng-class="{'oui-field_error': $ctrl.softphoneAddDeviceForm.input_name.$invalid}"
+                            placeholder="{{'telephony_line_softphone_device_name_placeholder'  | translate : {deviceNumber: '1'} }}"
+                            required
+                        />
+                        <div class="input-group-btn">
+                            <oui-button
+                                on-click=":: $ctrl.checkInputAndGenerateLink()"
+                                data-variant="primary"
+                            >
+                                <span
+                                    data-translate="telephony_line_softphone_generate_link"
+                                >
+                                </span>
+                            </oui-button>
+                        </div>
```

Comment:
You can simplify i think:
https://ovh.github.io/ovh-ui-kit/?path=/docs/design-system-components-input-native-group--validation